packages/@aws-cdk/cx-api/lib/features.ts

```diff
@@ -39,4 +25,24 @@ export const ENABLE_STACK_NAME_DUPLICATES_CONTEXT = '@aws-cdk/core:enableStackNa
  *
  * Use `cdk diff --fail` to exit with 1 if there's a diff.
  */
-export const ENABLE_DIFF_NO_FAIL = 'aws-cdk:enableDiffNoFail';
+export const ENABLE_DIFF_NO_FAIL_CONTEXT = 'aws-cdk:enableDiffNoFail';
```

Comment:
I like the alignment of naming to a convention. Should we also add that to `CONTRIBUTING`.
I imagine it's probably self explanatory when adding a key to this file

Reply:
I think it's self explanatory from this file.